README.md

```diff
@@ -20,15 +20,17 @@ Run `npm install`
 
```

Comment:
- [ ] It would be nicer to add some steps on how the user can run your project locally (ex: npm start or npm run start).